web/profiles/webspark/webspark/webspark.install

```diff
@@ -524,3 +524,19 @@ function webspark_update_10004(&$sandbox) {
     \Drupal::state()->set('configuration_locked', TRUE);
   }
 }
+
+/**
+ * WS2-2011: Set TTL to 1 hr
+ */
+function webspark_update_10005(&$sandbox) {
+  \Drupal::state()->set('configuration_locked', FALSE);
+  $config_factory = \Drupal::configFactory();
+  $config = $config_factory->getEditable('system.performance');
+  // Only set if default of none is configured. Otherwise, we assume the site
+  // owner has intentionally changed the setting.
+  if ($config->get('cache.page.max_age') !== '0') {
+    $config->set('cache.page.max_age', '3600');
+  }
```

Comment:
The logic here doesn't match the comment. According to the code, anything that isn't a zero value will get changed, but the comment says only to change it "if default of none is configured." To my mind, that means that the code should be `if ($config->get('cache.page.max_age') === '0')`.

Reply:
@tbutterf I'm becoming my parents - "Do as I say, not as I do." 😆 

In my initial testing this wasn't working - I thought it was because of string/int issues so I switched the logic to string and it worked... which was not what we want, but worked because of the logic being wrong AND Drupal doing type conversion with the values when it saved then... This is why people like strongly typed languages! 

With this fix, I've set the value to do a non-typed check (falsey check) so we account for if there's a null value as well as int zero or string zero. Any of those should trigger the value getting set. Then additionally, I've set the value saved back to an int to match what it actually should be.